src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  padding: $calendar-inner-padding;
+  max-width: 726px;
+  box-sizing: border-box;
```

Comment:
make sense to move to `*` selector to prevent code duplicating

Reply:
fixed it